apps/condo/domains/billing/tasks/sendNewBillingReceiptFilesNotifications.js

```diff
@@ -134,19 +134,26 @@ async function sendNewBillingReceiptFilesNotifications ({ organizationId, organi
         const uniqKey = `new_receipts_files_${contactId}_${filesIds}`
 
         // send message
-        await sendMessage(context, {
-            to: { email },
-            type: BILLING_RECEIPT_FILE_ADDED_TYPE,
-            uniqKey,
-            meta: {
-                dv: 1,
-                data: {
-                    organization: organizationName,
+        try {
+            await sendMessage(context, {
+                to: { email },
+                type: BILLING_RECEIPT_FILE_ADDED_TYPE,
+                uniqKey,
+                meta: {
+                    dv: 1,
+                    data: {
+                        organization: organizationName,
+                    },
+                    attachments,
                 },
-                attachments,
-            },
-            sender,
-        })
+                sender,
+            })
+        } catch (error) {
+            logger.error({
+                msg: `Failed to send message to ${email}`,
+                error,
+            })
+        }
         notificationsSent++
```

Comment:
Maybe move this counter into try block?

Reply:
nope, since I wanna to get number of really sent mails